### ISSUE TITLE
`serre-test` failure

### ISSUE THREAD

**Issue by gheber** [gheber/kenzo#141]
```
 Failure Details:
 --------------------------------
 RIGHT-SERRE-EFHM []: 
      Unexpected Error: #<UNDEFINED-FUNCTION RIGHT-SERRE-EFHM {1002821EB3}>
The function CAT-9:RIGHT-SERRE-EFHM is undefined...
 --------------------------------
 --------------------------------
 BROWN-REDUCTION []: 
      Unexpected Error: #<UNDEFINED-FUNCTION BROWN-REDUCTION {10028169E3}>
The function CAT-9:BROWN-REDUCTION is undefined...
 --------------------------------
 --------------------------------
 FIBRATION-DTAU-D []: 
      Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D {100280C703}>
The function CAT-9:FIBRATION-DTAU-D is undefined...
 --------------------------------
 --------------------------------
 FIBRATION-DTAU-D-INTR1 []: 
      Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D-INTR {1002800903}>
The function CAT-9:FIBRATION-DTAU-D-INTR is undefined...
 --------------------------------
 --------------------------------
 FIBRATION-DTAU-D-INTR []: 
      Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D-INTR {10027EE983}>
The function CAT-9:FIBRATION-DTAU-D-INTR is undefined...
 --------------------------------
```

**Comment by sphyynx** [comment]
These  functions RIGHT-SERRE-EFHM,  BROWN-REDUCTION, FIBRATION-DTAU-D,
FIBRATION-DTAU-D-INTR1 and FIBRATION-DTAU-D-INTR,  are neither defined
nor called in Kenzo-9.

Le 21/05/2019 02:16, Gerd Heber a écrit :
 >  Failure Details:
 >  --------------------------------
 >  RIGHT-SERRE-EFHM []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION RIGHT-SERRE-EFHM 
{1002821EB3}>
 > The function CAT-9:RIGHT-SERRE-EFHM is undefined...
 >  --------------------------------
 >  --------------------------------
 >  BROWN-REDUCTION []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION BROWN-REDUCTION 
{10028169E3}>
 > The function CAT-9:BROWN-REDUCTION is undefined...
 >  --------------------------------
 >  --------------------------------
 >  FIBRATION-DTAU-D []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D 
{100280C703}>
 > The function CAT-9:FIBRATION-DTAU-D is undefined...
 >  --------------------------------
 >  --------------------------------
 >  FIBRATION-DTAU-D-INTR1 []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D-INTR 
{1002800903}>
 > The function CAT-9:FIBRATION-DTAU-D-INTR is undefined...
 >  --------------------------------
 >  --------------------------------
 >  FIBRATION-DTAU-D-INTR []:
 >       Unexpected Error: #<UNDEFINED-FUNCTION FIBRATION-DTAU-D-INTR 
{10027EE983}>

**Comment by gheber** [comment]
Fixed in e519e58.
